cmd/cat-main.go

```diff
@@ -105,6 +162,13 @@ func catOut(r io.Reader, size int64) *probe.Error {
 	var n int64
 	var e error
 
+	// In case of a user showing the object content in a terminal,
+	// avoid printing control and other bad characters to avoid
+	// terminal session corruption
+	if terminal.IsTerminal(int(os.Stdout.Fd())) {
```

Comment:
> IMO we would need to introduce an option and work accordingly exactly like cat -A option

Looks like I can make an output similar to cat -A but I am not too much convinced how much would that be useful.

However, I don't think we need to add any new option flag: If the output is a terminal, it means the user wants to visually see that data, so I guess we can make the output prettier. But if the output is not a terminal, like `mc cat myminio/testbucket/object >/tmp/file` then we won't modify the output.

What do you think @balamurugana  ?